TOOLS/lua/autocrop.lua

```diff
@@ -25,74 +25,114 @@
 --
 -- to mpv's arguments. This may be desirable to allow cropdetect more
 -- time to collect data.
+
 require "mp.msg"
+require 'mp.options'
 
 script_name = mp.get_script_name()
 cropdetect_label = string.format("%s-cropdetect", script_name)
 crop_label = string.format("%s-crop", script_name)
 
--- number of seconds to gather cropdetect data
-detect_seconds = tonumber(mp.get_opt(string.format("%s.detect_seconds", script_name)))
-if not detect_seconds then
+-- auto: bool - Whether to automatically execute the script.
+-- auto_delay: number - Delay of automatic tasks, avoid certain videos starting with a black screen.
+-- auto_duration: number - Only videos longer than this length (seconds) will perform automatic tasks.
+-- auto_percent: number[1-100] - Only videos with a playback progress later than this will perform automatic tasks.
+-- detect_seconds: number - How long (seconds) to gather cropdetect data.
+-- detect_min_ratio: number[0.0-1.0] - The ratio of the minimum acceptable clip size to the original.
+-- detect_limits: number[0-255] - Black threshold for cropdetect.
```

Comment:
you could refer to the `limit` cropdetect parameter so users could look it up in ffmpeg docs for futher info.